tests/test_tutorial/test_commands/test_index/test_tutorial003.py

```diff
@@ -12,6 +12,7 @@
 
 def test_no_arg():
     result = runner.invoke(app)
+    assert result.exit_code == 0
     assert "[OPTIONS] COMMAND [ARGS]..." in result.output
     assert "Commands" in result.output
     assert "create" in result.output
```

Comment:
It would also be good to add a `assert not result.stderr` to ensure the doubled output isn't displayed